### DISCUSSION TITLE
PocketBase Roadmap

### DISCUSSION THREAD

**Discussion by ganigeorgiev** [pocketbase/pocketbase#123] [Announcements]
This was requested several times already, so here is a link to the roadmap - https://github.com/orgs/pocketbase/projects/2.

There you'll find a board with the planned features for v1.0.0 release, ordered by their current priority (no hard deadlines).

**Comment by zhenruyan** [comment]
This is exciting news. I will use it in the operation of millions of views

**Comment by ganigeorgiev** [comment]
Regarding requested time estimates via the support mail for features like #119, #311, #317 - unfortunately I'm not able to provide ETAs
because I work on PocketBase whenever I have free time (before/after work and over the weekends).

Furthermore, new features will have to be pushed back for a while until I manage to refactor and simplify the users management (#376).
The refactoring will take a while, so please be patient and if you are planning to use PocketBase for non-experimental application,
**please be aware that the changes will not be fully backward compatible and some manual migration steps will be necessary!**

**Comment by cadetpeters89** [comment]
This is awesome! Any idea how much funding you'd need to get to v1?

**Reply by ganigeorgiev** [reply]
It's not a matter of funding whether and when PocketBase will reach v1. It depends entirely on how long it'll take us to finalize the planned features.

Starting from November 14 I'll be able to [work fulltime on PocketBase for ~5months](https://github.com/pocketbase/pocketbase/discussions/730) but there are no hard ETAs yet so its hard to tell when v1 will be ready.

**Reply by cadetpeters89** [reply]
That makes sense (and congrats on the grant). I'll stay tuned and see what I can do to help. Good luck & keep it up, this project is very cool!

**Comment by florinutz** [comment]
pocketbase is awesome! I just wish the whole database would be decoupled so I could use my instance of postgres instead.

**Reply by thangngoc89** [reply]
@florinutz as Gani have explained multiple times, supporting another database would increase complexity. However, there is a fork called postgresbase if you’re interested.

**Reply by zhenruyan** [reply]
Thank you very much for recommending the branch I am maintaining, which has shown me that there are more people with the same needs as me
